org.eclipse.jdt.core.manipulation/common/org/eclipse/jdt/internal/ui/fix/CleanUpFixWrapper.java

```diff
@@ -28,13 +27,13 @@
 import org.eclipse.jdt.internal.ui.text.correction.ProblemLocation;
 
 /**
- * Class to wrap an ICleanUpFixCore and be used as an ICleanUpFix
+ * Class to wrap an ICleanUpFix and be used as an ICleanUpFix
```

Comment:
The comment doesn't make sense that now.

Reply:
I had wanted to wait until later, but I'm afraid I'll forget, honestly, so I'll add another commit now.  There's only like 15 references to the class and they are easily removed.

Reply:
Um, oops, seems I made it part of the part-2 commit instead of the part 3.   See https://github.com/eclipse-jdt/eclipse.jdt.ui/pull/1161